openstack/elektra/charts/postgres/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.backup.enabled }}
```

Comment:
Ahh, you are right i forgot to add the backup config directory for the backup container.